test/runtests.jl

```diff
@@ -563,10 +561,12 @@ end
            "message": "$message"
         }
         """
-    resp = HTTP.Messages.Response(400, body)
-    HTTP.setheader(resp.headers, "Content-Type" => "application/x-amz-json-1.1")
+    headers = ["Content-Type" => "application/x-amz-json-1.1"]
+    status_code = 400
 
-    ex = AWSException(HTTP.StatusError(400, resp))
+    req = HTTP.Request("GET", "http://therainforest.com", headers, body)
```

Comment:
Maybe add a comment that this won't actually connect to this server? Maybe consider using http://httpbin.org/ for testing

Reply:
Good idea. I'll write a comment explaining that this doesn't actually message the server.

I'm also updating this to properly be `https://amazon.ca`, as this test is for `AWSException()`, I feel like it should hit the appropriate website to be clear.